applications/RANSApplication/custom_conditions/data_containers/k_epsilon/epsilon_k_based_wall_condition_data.cpp

```diff
@@ -36,15 +36,28 @@ const Variable<double>& EpsilonKBasedWallConditionData::GetScalarVariable()
 }
 
 void EpsilonKBasedWallConditionData::Check(
-    const GeometryType& rGeometry,
+    const Condition& rCondition,
     const ProcessInfo& rCurrentProcessInfo)
 {
     KRATOS_TRY
-    const int number_of_nodes = rGeometry.PointsNumber();
+
+    const auto& r_geometry = rCondition.GetGeometry();
+    const auto& r_properties = rCondition.GetProperties();
+    const int number_of_nodes = r_geometry.PointsNumber();
+
+    KRATOS_ERROR_IF_NOT(rCurrentProcessInfo.Has(TURBULENT_ENERGY_DISSIPATION_RATE_SIGMA))
+        << "TURBULENT_ENERGY_DISSIPATION_RATE_SIGMA is not found in process info.\n";
+    KRATOS_ERROR_IF_NOT(rCurrentProcessInfo.Has(TURBULENCE_RANS_C_MU))
+        << "TURBULENCE_RANS_C_MU is not found in process info.\n";
+    KRATOS_ERROR_IF_NOT(rCurrentProcessInfo.Has(VON_KARMAN))
+        << "VON_KARMAN is not found in process info.\n";
+    KRATOS_ERROR_IF_NOT(r_properties.Has(RANS_LINEAR_LOG_LAW_Y_PLUS_LIMIT))
+        << "RANS_LINEAR_LOG_LAW_Y_PLUS_LIMIT is not found in condition properties [ Condition.Id() = "
+        << rCondition.Id() << ", Properties.Id() = " << r_properties.Id() << " ].\n";
 
     for (int i_node = 0; i_node < number_of_nodes; ++i_node)
```

Comment:
```suggestion
    for (std::size_t i_node = 0; i_node < number_of_nodes; ++i_node)
```
or better yet, a range-based for loop if `Geometry` supports it => `number_of_nodes` could be deleted

Reply:
done :)